src/main/java/org/sagebionetworks/bridge/udd/config/SpringConfig.java

```diff
@@ -70,9 +70,9 @@ public Table ddbSynapseMapTable() {
         return ddbClient().getTable(environmentConfig().get("synapse.map.table"));
     }
 
-    @Bean(name = "ddbUploadTable")
-    public Table ddbUploadTable() {
-        return ddbClient().getTable(ddbPrefix() + "Upload2");
+    @Bean(name = "ddbSynapseSurveyTable")
+    public Table ddbSynapseSurveyTable() {
+        return ddbClient().getTable(ddbPrefix() + "SynapseSurveyTables");
```

Comment:
This is a DDB table containing references to to a set of Synapse tables. This is confusing, and I'm not sure how to make it less confusing. Regardless, I changed the bean name to ddbSynapseSurveyTablesTable and added a comment to attempt to clarify things. (The DDB table name remains SynapseSurveyTables, to remain consistent-ish with our other DDB tables.)